pyaedt/edb.py

```diff
@@ -3356,3 +3358,147 @@ def calculate_initial_extent(self, expansion_factor):
         max_width = max_width * expansion_factor
         self.logger.info("The W factor is {}, The initial extent = {:e}".format(expansion_factor, max_width))
         return max_width
+
+    @pyaedt_function_handler()
+    def copy_zones(self, working_directory=None):
+        """Copy multizone EDB project to one new edb per zone.
+
+        Parameters
+        ----------
+        working_directory : str
+            Directory path where all EDB project are copied, if empty will use the current EDB project.
```

Comment:
For this one, too, you resolved the suggestion without making it. My suggestion follows what we set up as the standard format. Can you explain why you didn't accept the change.